cognee/tasks/documents/classify_documents.py

```
@@ -1,12 +1,17 @@
 from cognee.modules.data.models import Data
 from cognee.modules.data.processing.document_types import Document, PdfDocument, AudioDocument, ImageDocument, TextDocument
 
+EXTENSION_TO_DOCUMENT_CLASS = {
+    "pdf": PdfDocument,
+    "audio": AudioDocument,
+    "image": ImageDocument,
+    "pdf": TextDocument,
```

Ruff

Dictionary key literal `"pdf"` repeated

(F601)